service/internal/subjectmappingbuiltin/subject_mapping_builtin.go

```diff
@@ -187,7 +188,11 @@ ConditionEval:
 }
 
 func EvaluateCondition(condition *policy.Condition, entity map[string]any) (bool, error) {
-	mappedValues, err := ExecuteQuery(entity, condition.GetSubjectExternalSelectorValue())
+	flattenedEntity, err := flattening.Flatten(entity)
+	if err != nil {
+		return false, err
```

Comment:
Can we wrap these errors? 
```suggestion
		return false, fmt.Errorf("failure to EvaluateCondition in subject mapping builtin: %w", err)
```

Reply:
will do in second PR